Sources/MapboxNavigation/Banner.swift

```diff
@@ -1,27 +1,72 @@
 import UIKit
 
-// :nodoc:
+/**
+ A banner protocol provides the ability to show content inside `NavigationView`. Components that
+ conform to `Banner` protocol must be instances of the `UIViewController`. Banners are injected
+ into `BannerContainerView` and can have various properties:
+ - Can be shown at the top or at the bottom of the screen
+ - Can be expanded and collapsed
+ - Can have custom height
+ 
+ By default Mapbox Navigation SDK provides several default banners:
+ - `DestinationPreviewViewController` - banner that is shown at the bottom of the screen and allows
+ to show information about the final destination, preview available routes and start active navigation
+ session
+ - `RoutesPreviewViewController` - banner that is shown at the bottom of the screen and allows to
+ preview information about the current `Route` (expected travel time, distance and expected time of arrival)
+ - `BannerDismissalViewController` - banner that is shown at the top of the screen and allows to
+ dismiss already presented banner
+ */
 public protocol Banner: UIViewController {
     
+    /**
+     Configuration of the banner.
+     */
     var bannerConfiguration: BannerConfiguration { get }
 }
 
-// :nodoc:
+/**
+ Configuration of the banner that allows to change its default behavior.
+ */
 public struct BannerConfiguration {
     
-    // :nodoc:
+    /**
+     Position of the `Banner`. `Banner` is presented at the bottom of the screen by default.
+     */
     public private(set) var position: BannerPosition = .bottomLeading
     
-    // :nodoc:
+    /**
+     Initial height of the `Banner`. By default heights of the `Banner` are the following:
+     Top banner:
+     - Top safe area inset for both compact/regular-height size class
+     
+     Bottom banner:
+     - 80.0 + bottom safe area inset for regular-height size class
+     - 60.0 + bottom safe area inset for compact-height size class
+     */
     public private(set) var height: CGFloat? = nil
     
-    // :nodoc:
+    /**
+     A Boolean value that determines whether `Banner` can be expanded or not. Defaults to `false`.
+     Presented `Banner` is collapsed by default.
+     */
     public private(set) var isExpandable: Bool = false
     
-    // :nodoc:
+    /**
+     A floating-point value that denotes extent by which `Banner` can be expanded. Defaults to `0.0`.
```

Comment:
What is this extent measured in? Points? Percentage of the screen?

Reply:
Points, made this clarification (and in other places as well).